README.md

```diff
@@ -1 +1,103 @@
-# openapi-msw
+<h1 align="center">OpenApi-MSW</h1>
+
+A tiny, type-safe wrapper around [MSW](https://mswjs.io) to add support for full
+type inference from OpenApi schema definitions that are generated with
+[OpenApi-TS](https://openapi-ts.pages.dev/introduction).
+
+## Installation
+
+You can install OpenApi-MSW with this shell command:
+
+```bash
+npm i -D openapi-msw
+```
+
+**Note:** This package has a peer-dependency to MSW **v2**. There is no plan to
+provide backwards compatibility for MSW v1.
+
+## Usage Guide
+
+This guide assumes that you already have OpenApi-TS setup and configured to
+generate `paths` definitions. Please refer to
+[their setup guide](https://openapi-ts.pages.dev/introduction) before continuing
```

Comment:
For easy scannability, it's nice to have link names show completely where they're going. E.g. "OpenApi TS setup guide".